metaflow/plugins/conda/conda_environment.py

```diff
@@ -109,8 +109,9 @@ def get_client_info(cls, flow_name, metadata):
         if info is None or env_id is None:
             return {'type': 'conda'}
         info = json.loads(info)
-        with cls._filecache.get_data(info['ds_type'], flow_name, info['location'], info['sha']) as f:
-            with tarfile.open(fileobj=f, mode='r:gz') as tar:
+        with cls._filecache.get_data(info['ds_type'], flow_name, info['location'], info['sha']) as data:
```

Comment:
I actually don't think we need the with here, it's just _, blob = cls._filecache and then blob = BytesIO(blob) (I think)

Reply:
Made changes based on your suggestion